modules/n1ql/pages/n1ql-language-reference/limit.adoc

```diff
@@ -1,4 +1,5 @@
 = LIMIT clause
+:imagesdir: ../../assets/images
```

Comment:
This was an unrelated driveby fix?

Reply:
This attribute is purely for the benefit of authors. It means you get to see the images when you preview the document as a standalone AsciiDoc file, e.g. in your browser or in GitHub. It is stripped out / ignored by Antora. It's not conventional to add this attribute but I find it useful.

Reply:
Nice! Yes, I already took advantage of that by clicking through to the github preview, so definitely useful!